### ISSUE TITLE
Planetscale Driver doesn't work anymore after updating to 5.6.0 

### ISSUE THREAD

**Issue by cstrnt** [prisma/prisma#21956]
### Bug description

I've updated all my dependencies (`@prisma/adapter-planetscale`, `@prisma/client` and `prisma`) to v5.6.0.

I'm now getting an error that the given PlanetScale client isn't an instance of of the class `Client` from `@planetscale/database`

The error is thrown here https://github.com/prisma/prisma/commit/29bd6252d2692c300fa2d81024837c5eff2b3f2f because the `instanceof` check fails. I've checked that I have the same version of `@planetscale/database` locally installed

### How to reproduce

1. Update given packages to 5.6.0
2. Install all packages with pnpm v8.6.9 (should potentially break with other versions as well)
3. Update the code according to the migration guide (https://github.com/prisma/prisma/releases/tag/5.6.0)
4. Breaks

### Expected behavior

Prisma should work. The error is invalid since the `Client` instance I'm passing to `PrismaPlanetScale` is in-fact an instance of the Client class.

### Prisma information

```prisma
// Add your schema.prisma
```

```ts
// Add your code using Prisma Client
```

### Environment & setup

- OS: MacOS
- Database: MySql / Vitesse
- Node.js version: v18.17.0
- Package Manager: pnpm v8.6.9

### Prisma Version

```
5.6.0
```

**Comment by SevInf** [comment]
Can confirm the issue, working on a fix.

**Comment by SevInf** [comment]
Workaround for now: use common.js import for `@planetscale/database` :

```ts
import { createRequire } from "node:module";
import { PrismaPlanetScale } from "@prisma/adapter-planetscale";

const require = createRequire(import.meta.url);
const { Client } = require("@planetscale/database");

const adapter = new PrismaPlanetScale(
  new Client({ url:  process.env.DATABASE_URL  })
);
```

**Comment by cstrnt** [comment]
Can confirm that this works: https://github.com/tryabby/abby/pull/100

**Comment by SevInf** [comment]
Hi @cstrnt.
Thank you for confirming the workaround. We are also opened a PR with a proper fix. Could we ask you to verify that `5.7.0-integration-fix-export-map.3` version of `@prisma/adapter-planetscale` works for you with ESM import too? We noramally don't recommend using dev snapshots in production, but it will help us to verify the fix. At the moment, this is the code from `5.6.0` plus the fix for the issue.

**Comment by michaelhays** [comment]
I saw #21962 was reverted [here](https://github.com/prisma/prisma/commit/d4f2f3be1704286292bb727f0e4f38930e1c4fc6); should this issue be re-opened?

**Comment by SevInf** [comment]
@michaelhays your second link points to commit with a fix, not the revert.
If you mean [this commit](https://github.com/prisma/prisma/commit/0fd3620e43efff63f0c81468e779351bbb3322de) instead - it is on different branch that will not ever get merged into `main`. It was needed just for a few hours to unblock work on a different repo and none of the published dev snapshots contained the revert.

**Comment by michaelhays** [comment]
Ah yep my bad, I linked to the wrong one there.

Okay awesome, thanks for the info!
